topostats/io.py

```diff
@@ -497,9 +497,9 @@ def load_spm(self) -> tuple:
             # trying to return the error with options of possible channel values
             labels = []
             for channel in [layer[b"@2:Image Data"][0] for layer in scan.layers]:
-                channel_name = channel.decode("latin1").split(" ")[1][1:-1]
-                # channel_description = channel.decode('latin1').split('"')[1] # incase the blank field raises quesions?
-                labels.append(channel_name)
+                # channel_name = channel.decode("latin1").split(" ")[1][1:-1]
```

Comment:
Do we need to retain this line if the correct value should always be `channel_description`?

If not this line can be removed.

The code will still be present in the Git history and referenceable by looking at the Git blame for the line that remains, tracing it back to the commits and then its possible to see what changed if the old value is required.